### PR TITLE
OO: Add inline labels to stacked bar charts

### LINKED ISSUE #55: Add inline labels to stacked bar charts

### PR DESCRIPTION
**Pull request by okal**
Closes #55

**Note to reviewer**: Create release when merging.

### CODE REVIEW

File: src/vega_viewer/vega/specs/stacked_horizontal_bar_chart.cljs [hunk 1]
```diff
@@ -68,6 +72,17 @@
                                         :field "group"}}
                          :update {:fillOpacity {:value 1}}
                          :hover {:fillOpacity {:value 0.9}}}}
+           {:type "text"
+            :from {:data "grouped-data"}
+            :properties {:enter {:y {:scale "y"
+                                     :field "category"
+                                     :offset (-> bar-height
```
Comment (pld) [inline src/vega_viewer/vega/specs/stacked_horizontal_bar_chart.cljs:79]: what's the reasoning behind these magic numbers?

Comment (okal) [inline src/vega_viewer/vega/specs/stacked_horizontal_bar_chart.cljs:79]: It's how to center the bit of text vertically within the bar, in a way that will accommodate any possible future changes to the bar height. You're right, I'll pull them out.

Comment (okal) [inline src/vega_viewer/vega/specs/stacked_horizontal_bar_chart.cljs:79]: i.e. half of the bar height + an offset.

Comment (pld) [inline src/vega_viewer/vega/specs/stacked_horizontal_bar_chart.cljs:79]: got it, cool

**Comment by okal** [comment]
Thought I was going to need to change that?

**Comment by okal** [comment]
Also needed a release, per the description.

**Comment by okal** [comment]
But I can do that outside of this.

**Comment by pld** [comment]
Yeah, I guess it would be better abstracted as a function

**Comment by okal** [comment]
Why a function? It won't take any arguments and will always return the same thing.

**Comment by pld** [comment]
`bar-height` would be the argument, and this would give you a way to document what the calculation is doing and isolate it. This is only a suggestion, how are you think it would change?

**Comment by okal** [comment]
It's tied to the bar height which is (currently) a constant - `vega-viewer.vega.specs.constants.bar-height`. Would have just done a `def` with a docstring.

**Comment by pld** [comment]
Cool, I think you should do that

**Pull request merged**